src/database/song_segment.py

```diff
@@ -0,0 +1,52 @@
+from database.database import Database
+from database.storinator import Storinator
+
+
+class SongSegment(Storinator):
+    def __init__(self):
+        self._dbname = 'song_segmentation'
```

Comment:
This should be *col* or something similar as it is the name of the collection, not the database ;)